src/riscv/lib/src/machine_state/page_cache.rs

```diff
@@ -21,6 +21,7 @@ pub(crate) mod code_page_entry;
 pub(crate) mod dispatch;
 pub(crate) mod interpreted;
 pub(crate) mod jitted;
+pub(crate) mod router;
```

Comment:
```suggestion
mod router;
```
This shouldn't need to be referenced from anywhere other than the `page_cache` itself.